src/cdp/DefaultNetwork.ts

```diff
@@ -59,6 +59,7 @@ export class DefaultNetwork implements Network {
       ]);
     } catch (e) {
       // ADHOC: handle any unforeseen issues while detaching from targets.
+      this.logger.debug(`Unforeseen error while detaching from targets: ${e}`);
```

Comment:
```suggestion
      this.logger.debug(`Unexpected error while detaching from targets: ${e}`);
```

Reply:
I don't think so, this error has no impact on the flow or results.